ironfish/src/wallet/wallet.ts

```diff
@@ -882,7 +878,7 @@ export class Accounts {
         )
 
         if (isExpired) {
-          await this.deleteTransaction(transaction)
+          await this.expireTransaction(transaction)
```

Comment:
This will keep re-expiring the same transactions over and over for eternity We should add have a separate list of TX that need to be checked for expiration that shrinks over time. Maybe a new composite key table that's `[Account.id, expirationSequence, TransactionHash]`,

Reply:
Didn't think of that 🤦 

Having a separate list of pending transactions will probably be beneficial for larger wallets anyway to avoid repeatedly iterating over all transactions